operations/rates.md

```diff
@@ -154,7 +195,48 @@ The price in the response is dependent on the enterprise's [pricing](configurati
     "CategoryPrices": [
         {
             "CategoryId": "e3aa3117-dff0-46b7-b49a-2c0391e70ff9",
-            "Prices": [ 20, 20, 20 ]
+            "AmountPrices": [
```

Comment:
Same as comment above - `Prices` is being replaced by `AmountPrices` and therefore it needs an entry in the **Deprecations** table.